main/getting-started/before-using-agoric.md

```diff
@@ -121,3 +122,9 @@ To check that it's installed correctly:
 ```shell
 agoric --version # v0.16.0 "community-dev" branch
 ```
+
+If the install was successful, you are ready to proceed to [starting a project](/getting-started/start-a-project.md).
+
+Should `agoric` not be found, then you may need to add the parent directory of where your `yarn link-cli` command created a "agoric" directory.
+
+Please see the [Linux add to \$PATH: Fix "command not found" error (Linux & Mac) - YouTube](https://www.youtube.com/watch?v=gkqsLRDnqlA) for more information.
```

Comment:
It's a little odd that we discuss adding to PATH only in the quickstart section. The premise of the quickstart section is that it's for people who know the basics.

Adding a `export PATH=~/bin:$PATH` line in the code box seems more straightforward. Then we could omit these two paragraphs or move them down where we have "Note: Run echo $PATH to see your current $PATH."

not critical I suppose.

Reply:
thanks for approving the PR! regardless of the debate I think it's in superior condition! =) 

so if look we look at the greats like installing Node.js then it automatically links for us. Also, our VS Code extension is handling it beautifully. Ideally, I'd like to see us get to a place in which the `yarn link-cli` command is optional. 

That said, the instructions with an `export PATH=~/bin:$PATH` creates a new "bin" directory and adds more complexity to the $PATH. It's messy. After doing this several times, I think running `echo $PATH`, finding your local bin, and adding `agoric` in that location is optimal imo.

Reply:
> After doing this several times, I think running echo $PATH, finding your local bin, and adding agoric in that location is optimal imo.

Then shouldn't the quickstart section do likewise?